source/index.html.md

```diff
@@ -1383,3 +1383,90 @@ curl --request POST \
 Name | Data Type | Description
 -------------- | -------------- | --------------
 ORDER_ID | String | The ID of the order to acknowledge
+
+# Users
+
+## GET User
+
+Show user for the specified ID.
+
+Url format:
+
+/api/users/USER_ID
+
+```shell
+curl -X GET \
+  https://app.sellerchamp.com/api/users/61450d5c7305cf1e6583ef50.json \
+  -H 'content-type: application/json' \
+  -H 'token: YOUR-TOKEN' 
+}'
+
+Make sure to replace `YOUR-TOKEN` with your API key token.
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+  "user": {
+    "id": "61450d5c7305cf1e6583ef50",
+    "api_token": "c0bbf17e87440d1e1fdc497c9ea3b9ec",
```

Comment:
This is fake data or real data that is being exposed here?

Reply:
Also, dont send api token in the response. That's like sending password in cleartext.

Reply:
Data is from my local.
however, I updated it to dummy ones now.